data/json/overmap/specials.json

```diff
@@ -57,13 +57,11 @@
             { "point":[-1,1,0], "overmap": "motel_2_north"},
             { "point":[0,1,0], "overmap": "motel_3_north"}
         ],
-        "connections" : [
-            { "point" : [0,-1,0] }
-        ],
+        "connections": [ { "point": [ 0, -1, 0 ], "terrain": "road", "existing": true } ],
         "locations" : [ "land", "swamp" ],
-        "city_distance" : [10, 120],
-        "city_sizes" : [1, 12],
-        "occurrences" : [0, 5],
+        "city_distance" : [8, 120],
+        "city_sizes" : [3, 12],
```

Comment:
Motels IRL often show up near even very small centers, I think this could probably stay small. Is there something in particular you wanted to prevent happening here?